package.json

```diff
@@ -84,11 +85,13 @@
 		"chokidar": "^3.3.1",
 		"eslint": "^6.8.0",
 		"eslint-config-bamboo": "^3.0.0",
+		"jest": "^25.1.0",
+		"ts-jest": "^25.2.0",
 		"typescript": "^3.7.5"
 	},
 	"_moduleAliases": {
-		"@utils": "dist/src/lib/util",
-		"@lib": "dist/src/lib",
+		"@utils": "dist/lib/util",
+		"@lib": "dist/lib",
```

Comment:
Why was this needed? It seemed correct the way it was previously. Mind explaining so it is visible if any other reviews request it?

Reply:
Because TSC, when suggesting those paths, it doesn't do in order of length, but in key insertion order.

By placing `@utils` before `@lib`, it'll suggest `@utils/util.ts` before `@lib/utils/util.ts`, and automatic import will take the first one.

Reply:
Unless you meant that it was `dist/src/lib` before, in which case it's because with this change there is no `src` directory in `dist` anymore as the `rootDir` TSC config option has been set.

Reply:
That is what it was directed it. So I ask why was dist/src/ moved to just dist/